native/cocos/bindings/jswrapper/v8/Object.cpp

```diff
@@ -163,6 +164,54 @@ Object *Object::createPlainObject() {
     return obj;
 }
 
+std::unordered_map<Object*, v8::Persistent<v8::Promise::Resolver>*> Object::resolverMap;
+
+void Object::resolverPromise(Object *object, const Value &value) {
+    auto it = resolverMap.find(object);
+    if (it != resolverMap.end()) {
+        v8::Isolate *isolate = __isolate;
```

Comment:
double underscore+lowercase variable is reserved by c++. Please change __isolate later.